apps/src/code-studio/components/ToggleSwitch.jsx

```diff
@@ -10,22 +10,32 @@ class ToggleSwitch extends React.Component {
     isToggledOn: PropTypes.bool.isRequired,
     onToggle: PropTypes.func.isRequired,
     label: PropTypes.string.isRequired,
+    expands: PropTypes.string,
```

Comment:
wasn't sure what to call this prop, feel free to suggest alternatives

Reply:
Is this supposed to represent that the toggle expands a section?

Reply:
I also cannot think of a better name, so I'm okay with this

Reply:
it's both "this toggle `expands` the element with the given ID", and sort-of doubling as a boolean to add/remove the aria-attributes if it exists